packages/landing/src/config.map.ts

```diff
@@ -145,7 +145,11 @@ export class MapConfig extends Emitter<MapConfigEvents> {
     this.style = style ?? null;
     this.layerId = layerId.startsWith('im_') ? layerId.slice(3) : layerId;
     this.tileMatrix = tileMatrix;
-    this.labels = labels;
+    if (labels == null) {
+      this.labels = layerId === 'aerial' && !this.debug.debug;
+    } else {
+      this.labels = labels === 'true';
```

Comment:
Since we want this to mostly work like a flag, something more like `?labels` should work

```
x =  new URLSearchParams("?labels")
x.get('labels') // ""
```

I think `!== 'false'` would be better.